src/sass/_style.scss

```diff
@@ -290,22 +289,22 @@ select.form-control {
 
 #password-requirement-left {
   opacity: 1;
-  @extend .x-small;
+  font-size: var(--pgn-typography-font-size-xs) !important;
   filter: drop-shadow($elevation-level-2-shadow) drop-shadow($elevation-level-2-shadow) !important;
   right: 0.2rem !important;
   .tooltip-inner {
-    background: $white;
+    background: var(--pgn-color-white);
     display: block;
-    color: $gray-500;
+    color: var(--pgn-color-gray-500);
   }
   .arrow::before {
-    border-left-color: $white;
+    border-left-color: var(--pgn-color-white);
   }
 }
 
 #password-requirement-top {
-  @extend .x-small;
-  filter: drop-shadow($elevation-level-2-shadow) drop-shadow($elevation-level-2-shadow) !important;
+  font-size: var(--pgn-typography-font-size-xs) !important;
```

Comment:
It looks like `.x-small` is still listed in css utilities on the docs site https://paragon-openedx.netlify.app/foundations/css-utilities/

Does continuing to use `.x-small` here break things?

Reply:
This change is good as-is (see https://github.com/openedx/frontend-app-authn/pull/1436#discussion_r2079869474 for context)